webplugin/js/app/kommunicate-client.js

```diff
@@ -87,7 +90,9 @@ Kommunicate.client = {
             GROUP_CREATION_URL: window.kommunicate.IFRAME_OVERRIDES ? window.kommunicate.IFRAME_OVERRIDES.GROUP_CREATION_URL : parent.location.href,
             conversationMetadata:JSON.stringify(conversationDetail.metadata),
         };
-    
+        if (kommunicateSettings?.KM_TEST_PAGE) {
```

Comment:
we can't have this code in the chat widget, figure out a way to add it through the website test page

Reply:
we are passing KM_TEST_PAGE key from website test page in the defaultSetting and here we are capturing it.

Reply:
i don't think this is the right way to do, there must be a way to achieve the same from website page javascript itself without touching chat widget code.
if required take help from @anujpanwar01

Reply:
this is weird, @anujpanwar01  is this not possible to pass the key: value pair as used in standard way